### ISSUE TITLE
Update to Go 1.20 and remove deprecated dependencies

### ISSUE THREAD

**Issue by kamikazechaser** [iancmcc/bingo#1]
```
# github.com/iancmcc/bingo.test
github.com/iancmcc/bingo/bytes.InvertArrayLarge: relocation target crypto/cipher.xorBytes not defined
FAIL	github.com/iancmcc/bingo [build failed]
FAIL
 ```

XorBytes is now exported by the stdlib. I can send a PR updating the go.mod version directive to 1.20 and replacing the xorBytes fn.

**Comment by zhangbo2008** [comment]
how to solve

**Comment by iancmcc** [comment]
Fixed in 0.2.0
